context/app/static/js/components/home/Hero/const.tsx

```diff
@@ -13,6 +13,20 @@ const timelineIconProps = {
 } as const;
 
 export const HOME_TIMELINE_ITEMS: TimelineData[] = [
+  {
+    title: 'Unified Dataset Pages Now Available',
+    titleHref: '/search?entity_type[0]=Dataset',
```

Comment:
Could it make more sense to link to a specific dataset?

Reply:
Tiffany and I concurred on linking to the search page since this is a pretty global change.

Reply:
Yes I was thinking dataset search page since all of the dataset page designs should be impacted by this.